### DISCUSSION TITLE
Seriously annoying issue with web radios

### DISCUSSION THREAD

**Discussion by cmh714** [rern/rAudio#483] [Fixed Issues]
OK, Ive been having this issue for the last few months and was hoping it would resolve itself magically but it hasnt....I am very long time user and GREATLY appreciate all the work @rern has put into making things better!!!

The issue is web radio streams over 128k have been causing grief for the last few months as I already stated. 128k streams play fine no issues. My streams that are 256 or 750+ now have issues. When the track changes, I lose the radio and it jumps down to the next selection in the playlist. Nothing has changed on my side. Pi is wired, speed has never been an issue so not sure what is causing my problems....fully updated both for raudio and archlinux. and like I said, I have been using these for a long time....

Here are 3 stations all from the same source....Zappa Radio
128k: http://streams.norbert.de:8000/zappa.mp3
256k: http://streams.norbert.de:8000/zappa.aac
750+: https://streams.norbert.de/zappa.flac

I have absolutely no issues streaming HD music from my NAS so its only the radio streams....

Any suggestions, especially from @rern would be most appreciated especially if they work!!! :)
TIA!

**Comment by rern** [comment]
I've tested and yes it's an issue.

It could be this:
(from [norbert.de](https://www.norbert.de/index.php?id=10)):
> ...
> ### 10.05.2021: Updated Feed Software
> The feed was not very reliable, the streams went offline once or twice a week. I also had the problem that it did not preserve the peak listener count, so I could not see how many people use the streams. I also had the feeling that some clients would get kicked out after each song. Not a pleasant listening experience. ...

**Comment by cmh714** [comment]
Not sure how I didnt see that before....but ok yes that seems to have fixed the 256 stream. The 750+ one is still hosed but I guess I should assume that is server side as well....Thank you @rern, again glad you keep this rocking!
